### PR TITLE
Update @actions/core to 1.10.0

### PR DESCRIPTION
**Pull request by rentziass**
This updates @actions/core to use the [file version](https://github.com/actions/toolkit/pull/1178) of `save-state` and
`set-output` workflow commands.

### CODE REVIEW

**Review by marko-zivic-93 — APPROVED** [review]

**Comment by giordano** [comment]
Would fix #589

**Review by theomart77 — APPROVED** [review]

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -10,13 +10,14 @@ on:
       - releases/*
     paths-ignore:
       - '**.md'
+  workflow_dispatch:
```
Comment (dmitry-shibanov) [inline .github/workflows/build-test.yml:13]: Could you please remove it.

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -10,13 +10,14 @@ on:
...
 
 jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest]
```
Comment (dmitry-shibanov) [inline .github/workflows/build-test.yml:20]: Could you please revert these changes.

**Review by dmitry-shibanov — COMMENTED** [review]
Could you please add `process.env['GITHUB_OUTPUT'] = '';` in beforeEach for installer.test.ts

**Comment by rentziass** [comment]
Thank you @dmitry-shibanov! It was end of day and I couldn't wrap my head around that failure, nor could reproduce it locally 😄 Reverted all the unnecessary changes I was using to debug this in my fork.

**Comment by dmitry-shibanov** [comment]
> Thank you @dmitry-shibanov! It was end of day and I couldn't wrap my head around that failure, nor could reproduce it locally 😄 Reverted all the unnecessary changes I was using to debug this in my fork.

You're welcome. I'm going to merge your pull request. Falling for yarn 1 is not related to your changes.

**Review by dmitry-shibanov — APPROVED** [review]

**Review by randydwhitephotography — COMMENTED** [review]
signed-off-by Randy D White randydwhite@randydwhite.com

**Comment by randydwhitephotography** [comment]
signed-off-by Randy D White randydwhite@randydwhite.com
